### PR TITLE
Tests for snackbar slice

### LINKED ISSUE #170: Increase test coverage for redux slices

### PR DESCRIPTION
**Pull request by OliaHrytsak**
image

And coverage for all files after latest changes
image

### CODE REVIEW

**Review by D1nviz — APPROVED** [review]

File: code/src/store/slices/snackbarSlice.test.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    expect(nextState.config).toEqual({ variant: "error", ...config });
+  });
+
+  test("should handle closeSnackbar", () => {
+    const prevState = { ...initialState, isOpen: true };
+    const nextState = snackbarReducer(prevState, closeSnackbar());
+    expect(nextState.isOpen).toBe(false);
+  });
+
+  test("should handle openSnackbarWithTimeout", async () => {
```
Comment (mhevyk) [inline code/src/store/slices/snackbarSlice.test.tsx]: Remove `async` here

File: code/src/store/slices/snackbarSlice.test.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    expect(store.getState().snackbar.isOpen).toBe(true);
+    expect(store.getState().snackbar.config.messageTranslationKey).toBe(
+      "Test message"
+    );
+
+    jest.advanceTimersByTime(1500);
+    expect(store.getState().snackbar.isOpen).toBe(false);
+  });
+
+  test("should handle openSnackbarWithTimeout with default timer", async () => {
```
Comment (mhevyk) [inline code/src/store/slices/snackbarSlice.test.tsx]: Same here

**Review by heshvab — APPROVED** [review]

File: code/src/store/slices/snackbarSlice.test.tsx (hunk 1 above)
Comment (OliaHrytsak) [reply code/src/store/slices/snackbarSlice.test.tsx]: Removed

File: code/src/store/slices/snackbarSlice.test.tsx (hunk 2 above)
Comment (OliaHrytsak) [reply code/src/store/slices/snackbarSlice.test.tsx]: Removed

File: code/src/store/slices/snackbarSlice.test.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  typeof useAppSelector
+>;
+
+jest.useFakeTimers();
+
+describe("snackbarSlice reducers", () => {
+  const initialState = {
+    isOpen: false,
+    config: {} as BaseSnackbarConfig
+  };
```
Comment (dmtrth25) [inline code/src/store/slices/snackbarSlice.test.tsx]: all mocked data should move out of `describe` block

File: code/src/store/slices/snackbarSlice.test.tsx [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+  afterEach(() => {
+    jest.clearAllMocks();
+    jest.clearAllTimers();
+  });
+
+  test("should return the initial state", () => {
+    expect(snackbarReducer(undefined, { type: "unknown" })).toEqual(
```
Comment (dmtrth25) [inline code/src/store/slices/snackbarSlice.test.tsx]: below you created variable for `snackbarReducer` expectation for other tests
do the same here

**Review by mhevyk — APPROVED** [review]

File: code/src/store/slices/snackbarSlice.test.tsx (hunk 4 above)
Comment (OliaHrytsak) [reply code/src/store/slices/snackbarSlice.test.tsx]: Done)

File: code/src/store/slices/snackbarSlice.test.tsx (hunk 3 above)
Comment (OliaHrytsak) [reply code/src/store/slices/snackbarSlice.test.tsx]: Moved)

**Review by dmtrth25 — APPROVED** [review]

**Pull request merged**